ui/src/components/SignIn/SignIn.jsx

```diff
@@ -0,0 +1,44 @@
+import GoogleLogin from 'react-google-login';
+
+export default function SignIn(props) {
+    const handleLogin = async googleData => {
+        const response = await fetch("http:://localhost:3001/api/v1/auth/google", {
```

Comment:
If you plan to deploy this on a CI/CD pipeline, this url will mostly likely need to be stored as an environment variable or something similar to that to support different urls depending on if it's your local dev server or production.